### PR TITLE
Add yarn.lock file

### PR DESCRIPTION
**Pull request by maheshschand**
Shifting to yarn. Fixing issue #176 .

### CODE REVIEW

**Comment by Harkishen-Singh** [comment]
@maheshschand thanks for showing your interest. However, the fronend is the only part that makes use of node_modules. This is inside the **dashboard/** in v1.0 and v1.1. These are presently executed with npm which needs a change to yarn, and not in the root folder as you have done here. Also, you need to mention in INSTALL.md the instructions to use yarn.

**Comment by maheshschand** [comment]
I will make necessary changes and make a new pull request

File: node_modules/.yarn-integrity [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (aquibbaig) [inline node_modules/.yarn-integrity]: Shouldn't node_modules be in `.gitignore` ?

**Comment by maheshschand** [comment]
OK. I will make necessary changes. Thanks for reviewing.

**Comment by maheshschand** [comment]
@aquibbaig @Harkishen-Singh Could some one please review this pull request?

**Comment by aquibbaig** [comment]
Thanks @maheshschand. This LGTM. Just one last change, can you please remove the changes of `v1.0` from this pull request. We'd like to see yarn only in `v1.1` 🎉

**Comment by maheshschand** [comment]
I made necessary changes. Please review it @aquibbaig .

File: dashboard/v1.1/INSTALL.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+1. Install NodeJS and yarn.
+2. Clone the repository or its fork via 
+   ```
+   git clone https://github.com/zairza-cetb/bench-routes
+   ```
+3. Navigate into the cloned repo 
+   ```
+   cd bench-routes
+   ```
```
Comment (ganeshpatro321) [inline dashboard/v1.1/INSTALL.md]: Hi @maheshschand, I dont think the instructions here are valid. Please refer to the `Makefile` in the root directory to clear your doubts. Please revert the changes in this file. Thanks.

Comment (maheshschand) [reply dashboard/v1.1/INSTALL.md]: OK.

File: Makefile [hunk 3]
```diff
@@ -8,39 +8,39 @@ build:
 	mv main bench-routes
 
 view-v1.0:
-	cd dashboard/v1.0/ && sudo npm start
+	cd dashboard/v1.0/ && sudo yarn start
```
Comment (ganeshpatro321) [inline Makefile]: I think we dont have yarn in v1.0 . Make the changes accordingly. @maheshschand

File: Makefile [hunk 4]
```diff
@@ -8,39 +8,39 @@ build:
...
 	cd dashboard/v1.1/ && prettier '**/*.tsx' --list-different
-	cd dashboard/v1.1/ && npm run test
-	cd dashboard/v1.1/ && npm run build
-	cd dashboard/v1.1/ && npm start &
+	cd dashboard/v1.1/ && yarn run test
+	cd dashboard/v1.1/ && yarn run build
+	cd dashboard/v1.1/ && yarn start &
 
 
 test-views-only:
```
Comment (ganeshpatro321) [inline Makefile]: You can make different commands for `test-views-only` for v1.0 and v1.1, as there is a clash between yarn and npm

Like `test-views-only-v1.0` and `test-views-only-v1.1`

Comment (maheshschand) [reply Makefile]: Sorry. I didn't see it. I just replaced every word with yarn. Thanks for reviewing @ganeshpatro321 .

**Comment by maheshschand** [comment]
@ganeshpatro321 I made necessary changes. Please review it.

**Comment by ganeshpatro321** [comment]
Looks good. Make sure all the updated commands are good and working. Have you tested them, if yes, this is good to merge. @maheshschand

**Comment by ganeshpatro321** [comment]
Please check why the build is failing. Might be due to one of makefile commands.

**Comment by maheshschand** [comment]
I am looking into it
